### ISSUE TITLE
select.form-control height shorter than input.form-control

### ISSUE THREAD

**Issue by ericfelipe99** [twbs/bootstrap#19749]
O don't speak english, i'm brazilian. 
The form in jsfiddle show the problem... when create a grid with select using "form-control" class, the small height of select make a column problem. 

https://jsfiddle.net/Ls5qre31/

bootstrap alpha-2

**Comment by twbs-lmvtfy** [comment]
Hi @ericfelipe99!

You appear to have posted a live example (https://fiddle.jshell.net/Ls5qre31/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 64, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 71, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 78, column 17 thru column 30: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 94, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 101, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 108, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 115, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 135, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 142, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 149, column 17 thru column 30: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 164, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 171, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 178, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 185, column 21 thru column 34: Bad value ````  ```` for attribute ```` for ```` on element ```` label ````: An ID must not be the empty string.
* line 64, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 71, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 78, column 17 thru column 30: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 94, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 101, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 108, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 115, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 135, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 142, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 149, column 17 thru column 30: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 164, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 171, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 178, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.
* line 185, column 21 thru column 34: The ```` for ```` attribute of the ```` label ```` element must refer to a non-hidden form control.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by RyanZim** [comment]
For some reason, `<input>` is rendered with a height of 24px, while `<select>` is rendered with a height of 22px in Firefox. Oddly, IE works as intended; I haven't checked other browsers. 
@ericfelipe99: what browser are you using?

By the way, ignore twbs-lmvtfy - that doesn't apply.

**Comment by ericfelipe99** [comment]
hi, 

thanks for reply. 

i'm using Opera 36, chrome 49.0.2623.112 and edge on windows 10 and fully updated. 
internet explorer don't opened jsfiddler. 

all html error is only label attributes, nothing that compromise layout like wrong dom.

Thanks, 
Eric

**Comment by RyanZim** [comment]
Any insight, @cvrebert?

**Comment by cvrebert** [comment]
Boiling this down to just the relevant part: https://jsfiddle.net/cvrebert/fL8fuqnL/1/

**Comment by cvrebert** [comment]
Duplicate of #17194. Please try searching more in the future.
